### PR TITLE
meta-yadro: Initial version of meta-vesnin layer

### PR DESCRIPTION
**Pull request by nest1ing**
VESNIN is a quad OpenPower P8 machine manufactored by YADRO.
The machine uses an AST2400 BMC SoC.

Signed-off-by: Alexander Filippov <a.filippov@yadro.com>

### CODE REVIEW

**Review by artemsen — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Different components of meta-ibm are under different licenses (a mix
```
Comment (AlexanderAmelkin) [inline LICENSE:1]: This is NOT `meta-ibm`.

File: meta-vesnin/recipes-kernel/linux/linux-aspeed/0001-ARM-dts-aspeed-Add-YADRO-VESNIN-BMC.patch [hunk 2]
```diff
@@ -0,0 +1,270 @@
+From 99c53c63a485a7cf929e0bae9e1db2e6a288153b Mon Sep 17 00:00:00 2001
+From: Alexander Filippov <a.filippov@yadro.com>
+Date: Tue, 19 Feb 2019 15:02:06 +0300
+Subject: [PATCH] ARM: dts: aspeed: Add YADRO VESNIN BMC
+
+VESNIN is an OpenPower machine with an Aspeed 2400 BMC SoC manufactured
+by YADRO.
```
Comment (AlexanderAmelkin) [inline meta-vesnin/recipes-kernel/linux/linux-aspeed/0001-ARM-dts-aspeed-Add-YADRO-VESNIN-BMC.patch:7]: I'd say:
> VESNIN is a quad OpenPower P8 machine manufactured by YADRO.
> The machine uses an AST2400 BMC SoC.

The same applies to the commit log in this pull request itself.

File: meta-vesnin/recipes-kernel/linux/linux-aspeed/0002-vesnin-remap-aspeed-uart.patch [hunk 3]
```diff
@@ -0,0 +1,41 @@
+From 85027d94696464269ae77702ead788a14fc8305f Mon Sep 17 00:00:00 2001
+From: OpenEmbedded <oe.patch@oe>
```
Comment (AlexanderAmelkin) [inline meta-vesnin/recipes-kernel/linux/linux-aspeed/0002-vesnin-remap-aspeed-uart.patch:2]: I guess this also must be:
`From: Alexander Filippov <a.filippov@yadro.com>`

**Review by AlexanderAmelkin — CHANGES_REQUESTED** [review]

File: meta-vesnin/recipes-kernel/linux/linux-aspeed/0001-ARM-dts-aspeed-Add-YADRO-VESNIN-BMC.patch (hunk 2 above)
Comment (nest1ing) [reply meta-vesnin/recipes-kernel/linux/linux-aspeed/0001-ARM-dts-aspeed-Add-YADRO-VESNIN-BMC.patch:7]: It has been sent to the Linux mainline with this commit message, so I propose keeping it as is.
But I will change the commit message in this pull request.

**Comment by nest1ing** [comment]
I don't understand why it closed after force push.

**Comment by nest1ing** [comment]
Seems, it lost link to master and could not be merged.
I open a new PR.

**Comment by AlexanderAmelkin** [comment]
Just for the reference, the new PR is #2

**Pull request closed without merging**